adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureParser.scala

```diff
@@ -308,7 +338,7 @@ class NarrowPeakParser extends FeatureParser {
 
     // skip empty or invalid lines
     if (fields.length < 3) {
-      if (log.isDebugEnabled) log.debug("Empty or invalid NarrowPeak line: {}", line)
+      log.warn("Empty or invalid NarrowPeak line: {}", line)
```

Comment:
Blank lines (and browser & track header lines) are valid in BED and NarrowPeak files, and encouraged between different tracks in the same file, so `log.warn` is probably too severe here and above.  See e.g. https://github.com/bigdatagenomics/adam/blob/master/adam-core/src/test/resources/dvl1.200.bed#L34